test/e2e/handler/nncp_parallel_test.go

```diff
@@ -0,0 +1,75 @@
+package handler
+
+import (
+	"fmt"
+	"sync"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstate "github.com/nmstate/kubernetes-nmstate/api/shared"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("NNCP with parallel set to true", func() {
+	Context("when applying a policy to matching nodes", func() {
+		progressConditions := []nmstate.Condition{
+			nmstate.Condition{
+				Type:   nmstate.NodeNetworkConfigurationEnactmentConditionProgressing,
+				Status: corev1.ConditionTrue,
+			},
+			nmstate.Condition{
+				Type:   nmstate.NodeNetworkConfigurationEnactmentConditionAvailable,
+				Status: corev1.ConditionUnknown,
+			},
+			nmstate.Condition{
+				Type:   nmstate.NodeNetworkConfigurationEnactmentConditionFailing,
+				Status: corev1.ConditionUnknown,
+			},
+			nmstate.Condition{
+				Type:   nmstate.NodeNetworkConfigurationEnactmentConditionMatching,
+				Status: corev1.ConditionTrue,
+			},
+		}
+		BeforeEach(func() {
+			node := nodes[0]
+			By("Create a policy")
+			updateDesiredState(linuxBrUp(bridge1))
+			By(fmt.Sprintf("Wait for %s progressing state reached", node))
+			enactmentConditionsStatusEventually(node).Should(ConsistOf(progressConditions))
+		})
+		AfterEach(func() {
+			By("Remove the policy")
+			deletePolicy(TestPolicy)
+			By("Reset desired state at all nodes")
+			resetDesiredStateForNodes()
+		})
+		It("should be progressing on multiple nodes in parallel", func() {
+			if !parallelRollout {
```

Comment:
We don't need this we are skipping at CI with -skip

Reply:
Right, removed